File: src/Tiantong.Wms.Client/src/Entities/Invoice.ts
export default class Invoice {
  id: number = 0

  name: string = ''

  specification: string = ''

  unit: string = ''

  quantity: number = 0

  price: number = 0

  amount: number = 0

  tax_rate: number = 13

  tax_amount: number = 0

  invoice_number: string = ''

  invoice_type: string = '增值税专用发票'

}

File: src/Tiantong.Wms/Tiantong.Wms.Api/src/Entities/Invoice.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Tiantong.Wms.Api
{
  [Table("invoices")]
  public class Invoice : Entity
  {
    public virtual string name { get; set; }

    public virtual string specification { get; set; }

    public virtual string unit { get; set; }

    public virtual int quantity { get; set; }

    public virtual double price { get; set; }

    public virtual double amount { get; set; }

    public virtual double tax_rate { get; set; }

    public virtual double tax_amount { get; set; }

    public virtual string type { get; set; } = "";

    public virtual string number { get; set; }

  }
}
